includes/server-timing/load.php

```diff
@@ -204,7 +219,7 @@ static function ( $hookname ) {
 		);
 	}
 
-	$value['output_buffering'] = ! empty( $value['output_buffering'] );
+	$value['output_buffering'] = (bool) $value['output_buffering'];
```

Comment:
interesting, I assume phpstan flagged this?

Reply:
Actually it's something I applied based on https://github.com/WordPress/performance/pull/1091#discussion_r1543908524